analysis/organize_data.qmd

```diff
@@ -28,7 +28,7 @@ gss <- read_fwf(here("data","data_raw","GSS.dat"),
 ## Code variables
 
 ```{r}
-#| label: Removing missing values
+#| label: assign-missing-values
```

Comment:
Good practice is that labels should be single words not sentences. I changed all of them accordingly.